sdk/keyvault/keyvault-certificates/CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 - Marked `ErrorModel` as deprecated. It was erronously exported publicly in 4.1 and should not be used. Please change the type to use `CertificateOperationError` instead.
 - Fixed a bug with `beginDeleteCertificate` and `beginRecoverDeletedCertificate` in which unknown service errors wouldn't bubble up properly to the end users.
+- Renamed the `KeyVaultCertificateId` to `KeyVaultCertificateIdentifier`, and exported a method to parse Key Vault Certificate Ids: `parseKeyVaultCertificateIdentifier`.
```

Comment:
/cc @ramya-rao-a for comment. I believe she favored the shorter `*Id`.

Reply:
The main driver for us to move to Identifier is whether Python moves to identifier or not. There have been cases in which JavaScript has needed to do uniquely different things, but if possible, we should try to align with at least the other dynamic language we mainly support.

cc: @mccoyp , @chlowell 

I know that Python’s PR is currently aiming to use Identifier, but does anybody remember in which cases we’ve decided to use *Id in Python over *Identifier? To see if there’s a solid case to use one over the other.

Reply:
That's why I CC'd @ramya-rao-a - I recalled a conversation with her about this. But per internal discussions, it sounds like she agrees to just be consistent with the other 3 tier 1 languages and use "Identifier" as a suffix.

Reply:
Charles and I discussed this this morning, and Python doesn't have a particular preference that we're aware of -- and for Key Vault, neither of us has a particular preference either. We did initially go with *Id when we added this feature, so I would think that the languages could align on that again if JS has a preference for it. Otherwise, I think *Identifier is fine

Reply:
Seems there's no strong preference and 3 tier 1 languages use "Identifier" so let's go with that if there's no reason to use "Id" idiomatically (sounds like there's not).

Note that this particular API had been reviewed by archboard earlier.